velox/substrait/TypeUtils.cpp

```diff
@@ -54,6 +54,12 @@ TypePtr toVeloxType(const std::string& typeName) {
   if (typeKind == TypeKind::VARCHAR) {
     return VARCHAR();
   }
+  if (typeKind == TypeKind::INTEGER) {
+    return INTEGER();
+  }
+  if (typeKind == TypeKind::BIGINT) {
```

Comment:
nit: using switch statement would be a bit more conventional and readable

Reply:
changed.